src/pages/home/report/ReportActionsList.js

```diff
@@ -228,12 +228,21 @@ function ReportActionsList({
             unreadActionSubscription.current = null;
         }
 
-        // Listen to specific reportID for unread event and set the marker to new message
-        unreadActionSubscription.current = DeviceEventEmitter.addListener(`unreadAction_${report.reportID}`, (newLastReadTime) => {
+        const resetUnreadMarker = (newLastReadTime) => {
             cacheUnreadMarkers.delete(report.reportID);
             lastReadTimeRef.current = newLastReadTime;
             setCurrentUnreadMarker(null);
             setMessageManuallyMarkedUnread(new Date().getTime());
```

Comment:
Is this still true?

In one of our cases, we mark the action as "read",  but in the code, we execute the expression `setMessageManuallyMarkedUnread(...)`.